### ISSUE TITLE
fix(CSS): Navigation CSS fails on older iOS 

### ISSUE THREAD

**Issue by kjon-life** [kjon-life/kjon-life#2]
**Describe the bug**
iOS does not recognize the nested CSS in styles.css

**To Reproduce**
1. Go use an older iPad
2. Open the website in any browser
3. The menu will be centered and stacked instead of centered and column

**Expected behavior**
Menu should be centered and horizontal
Menu should have pseudo-elements for visited, active, etc

**Screenshots**

**Desktop (please complete the following information):**
 - OS: iOS
 - Browser verified in Safari, Chrome, Brave
 - Version 16.7.5

**Smartphone (please complete the following information):**
 - Device: iPad Pro
 - OS: 16.7.5
 - Browser: Any browser
 - Version: Current

**Additional context**

**Comment by kjon-life** [comment]
```CSS
/* navigation 
   older iOS won't render nested CSS*/
header {
  position: fixed;
  top: 0;
  left: 0;
  width: 100%;
  background-color: #f5f3f5; /* Adjust the background color as needed */
  padding: 20px;
  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); /* Add a subtle shadow for depth */
  z-index: 999; /* Ensure the header appears above other elements */
  display: flex;
  flex-direction: column;
  align-items: center;
  
  img {
    max-width: 150px;
    height: auto;
    margin: 40px auto 0;
    padding: 1em;
  }
  
  ul {
    list-style-type: none;
    display: flex;
    margin: 0;
    padding: 0;
 
    li {
      &:not(:last-child) {
        margin-right: 2em;
      }

      a {
        text-decoration: none;
        color: #C88304; /* link color */
      }

      &:hover {
        color: #3F0071; /* link color on hover */
        text-decoration: underline; /* underline on hover */
      }

      &:visited {
        color: #582707; /* visited link color */
      }

      &:active {
        color: #F5F3F5; /* active link color */
      }
    }
  }
}

```
This will require a rewrite of the article selector too.

**Comment by kjon-life** [comment]
Closes #2
